std/array.d

```diff
@@ -2175,21 +2196,12 @@ if (isInputRange!RoR &&
         auto arr2 = "Здравствуй Мир Unicode".to!(T);
         auto arr = ["Здравствуй", "Мир", "Unicode"].to!(T[]);
         assert(join(arr) == "ЗдравствуйМирUnicode");
-        static if (autodecodeStrings)
-        {
-            static foreach (S; AliasSeq!(char,wchar,dchar))
-            {{
-                auto jarr = arr.join(to!S(' '));
-                static assert(is(typeof(jarr) == T));
-                assert(jarr == arr2);
-            }}
-        }
-        else
-        {
-            // Turning off autodecode means the join() won't
-            // just convert arr[] to dchar, so mixing char
-            // types fails to compile.
-        }
+        static foreach (S; AliasSeq!(char,wchar,dchar))
```

Comment:
space after comma

Reply:
This is copy-pasted from above, I didn't change anything.